CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-* @defenseunicorns/uds-core
+* @defenseunicorns/uds-cli
 
 # Additional privileged files
 /CODEOWNERS @jeff-mccoy @catsby @UncleGedd @daveworth
```

Comment:
should we also remove `@catsby` and `@UncleGedd` from this line?

Reply:
👍 updated in 1311009